docs/docs/cloud/how-tos/add-human-in-the-loop.md

```diff
@@ -1,38 +1,8 @@
-# Human-in-the-loop
+# Human-in-the-loop API
```

Comment:
We should retitle to make it clear that it's Server API rather than framework API